src/app/public/src/modules/footer/footer.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+
+import { StacheNavLink } from '../nav';
+
+import { StacheConfigService } from '../shared';
+
+const _get = require('lodash.get');
+
+@Component({
+  selector: 'stache-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss']
+})
+export class StacheFooterComponent implements OnInit {
+  public copyright: string;
+  public siteName: string;
+  public resourceLinks: StacheNavLink[];
+
+  constructor(private configService: StacheConfigService) { }
+
+  ngOnInit() {
```

Comment:
Need an access modifier here

Reply:
Yeah for sure, the code in this PR is not ready for review, it's mostly non functional at the moment anyway.  This was just to make the branch/style changes easier to see to get feedback. :) Thanks and good catch though!